admin_guide/securing_builds.adoc

```diff
@@ -22,12 +22,15 @@ xref:../architecture/core_concepts/builds_and_image_streams.adoc#custom-build[Cu
 builds] are inherently less safe than
 xref:../architecture/core_concepts/builds_and_image_streams.adoc#source-build[Source
 builds], given that they can execute any code in the build with potentially full
-access to the node's Docker socket.  xref:../architecture/core_concepts/builds_and_image_streams.adoc#docker-build[Docker
+access to the node's Docker socket, and as such are disabled by default.  xref:../architecture/core_concepts/builds_and_image_streams.adoc#docker-build[Docker
 build] permission should also be granted with caution as a vulnerability in the Docker build
 logic could result in a privileges being granted on the host node.
 
-By default, all users that can create builds are granted permission to use all
-build strategies (Docker, Source-to-Image, and Custom).
+By default, all users that can create builds are granted permission to use the
+Docker and Source-to-Image build strategies. The Custom build strategy can be
+enabled by a user with xref:../architecture/additional_concepts/authorization.adoc#roles[*cluster-admin*]
```

Comment:
Users with xref:../architecture/additional_concepts/authorization.adoc#roles[*cluster-admin*] privileges can enable the Custom build strategy, as referenced in...